internal/cmd/auth.go

```diff
@@ -75,7 +76,7 @@ func authenticationGRPC(
 	var (
 		register = grpcRegisterers{
 			public,
-			auth.NewServer(logger, store, auth.WithAuditLoggingEnabled(cfg.Audit.Enabled())),
```

Comment:
this seems weird that we would only want to enable audit logging in the auth server if `token:deleted` is enabled. why are we changing that here?

Reply:
@markphelps Yeah so this was a discussion some time ago in regards to tokens.

When tokens are deleted, the server is only provided the ID of the token which maps to the token in the database. We had two choices to make:

1. Make the middleware aware of the auth service
2. Move the adding of the event to the span in the auth service

@GeorgeMac and I decided it was a bit weird to make the middleware auth service aware, so we decided to go with this logic where only for token deleted events we will add the event to the span down in the auth service layer.

Reply:
@markphelps Oh maybe I can change the naming of that Option. It only is for `token:deleted` my bad.

Reply:
i guess i dont fully understand still. if audit logging is enabled then all events should be added to the span, its up to the audit event checker to then filter out which ones are sent to the sinks (at least that was my understanding)

So im not 100% on why we need to specific check for `checker.Check("token:deleted")` in the `grpc.go` command below

Reply:
Oh would you mind adding this event/noun to the audit README if it isn't already there

Reply:
@markphelps will do!